LondonBritishLibrary/orient/BLorient600.xml

```diff
@@ -58,55 +362,94 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                         <width>378</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="3" writtenLines="29" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a fine hand, about the end of the  <date>17th century</date>.
-                                    
+
                                 </desc>
-                                
+
                                 <date notBefore="1600" notAfter="1700" resp="PRS10171Wright"/>
```

Comment:
I guess the time span can be narrowed since the hand is dated to the end of the 17th century